statusHandler/presenterStatusHandler.go

```diff
@@ -0,0 +1,296 @@
+package statusHandler
```

Comment:
might move this file in statusHandler/presenter package

Reply:
Done.